src/features/image-preview.js

```diff
@@ -0,0 +1,30 @@
+var debug = require('../helpers/debug');
+
+var enablePreview = exports.enablePreview = function() {
+    /* CONFIG */
+    xOffset = -250;
```

Comment:
I added "var" but I am not sure what you mean by the first part. Can you explain?

Thanks.